tests/garage/torch/modules/test_gaussian_mlp_module.py

```diff
@@ -0,0 +1,368 @@
+import pickle
+from garage.torch.modules.gaussian_mlp_module import GaussianMLPModule
+import unittest
+from unittest import mock
+import torch
+from torch import nn
+from nose2.tools.params import params
```

Comment:
please see the [updated CONTRIBUTING.md](https://github.com/rlworkgroup/garage/pull/678) for conventions on imports. these external imports should use full paths.

Reply:
your imports also need to be grouped properly. this is also covered in CONTRIBUTING.md

Reply:
fixed